### PR TITLE
Fix simple variable usage and add check before removing tmp file

### PR DESCRIPTION
**Pull request by radoondas**
This PR:
- fixes usage of simple variable 
- add check for file before trying it to remove

### CODE REVIEW

File: dev-tools/import_dashboards.sh [hunk 1]
```diff
@@ -109,8 +109,8 @@ fi
 echo "Import dashboards,visualizations, searches and index pattern from ${DIR} to ${ELASTICSEARCH} in ${KIBANA_INDEX}"
 
 # Workaround for: https://github.com/elastic/beats-dashboards/issues/94
-$CURL -XPUT "$ELASTICSEARCH/$KIBANA_INDEX" > /dev/null 2>&1 
-$CURL -XPUT "$ELASTICSEARCH/$KIBANA_INDEX/_mapping/search" -d'{"search": {"properties": {"hits": {"type": "integer"},
+${CURL} -XPUT "$ELASTICSEARCH/$KIBANA_INDEX" > /dev/null 2>&1
+${CURL} -XPUT "$ELASTICSEARCH/$KIBANA_INDEX/_mapping/search" -d'{"search": {"properties": {"hits": {"type": "integer"},
```
Comment (tsg) [inline dev-tools/import_dashboards.sh]: Same should be done for `ELASTICSEARCH` and `KIBANA_INDEX` on the same lines?

Comment (radoondas) [inline dev-tools/import_dashboards.sh]: Right, missed them. Fix in a minute.

Comment (radoondas) [inline dev-tools/import_dashboards.sh]: Fixed.

**Comment by tsg** [comment]
Thanks for the contribution, @radoondas!

**Pull request merged**